application/src/sass/_footer.scss

```diff
@@ -0,0 +1,4 @@
+#footer .footer-meta .credit {
+  font-size: 16px; /* Override for the GOV.UK template style */
+  margin-bottom: 10px;
+}
```

Comment:
I hate these "no line terminator" things...

Reply:
The pre-commit thing was meant to auto fix these 🤔

Reply:
Not for `.scss` files though? I did have a setting in sublime text to auto-fix these. Not sure what happened to it. Will check.